Source/Widgets/Geocoder/Geocoder.js

```diff
@@ -70,6 +70,11 @@ value: searchText,\
 valueUpdate: "afterkeydown",\
 disable: isSearchInProgress,\
 css: { "cesium-geocoder-input-wide" : keepExpanded || searchText.length > 0 }');
+        textBox.addEventListener('focus', function() {
+            setTimeout(function() {
```

Comment:
Sounds good, thanks for checking. Can you add a comment to the code indicating the setTimeout is needed for Safari, this way someone doesn't accidentally remove it in the future.  Thanks